environments/dev/files/publishing-requirements.txt

```diff
@@ -5,9 +5,10 @@
 
 -e git+https://github.com/Connexions/openstax-accounts.git#egg=openstax-accounts
 -e git+https://github.com/Connexions/cnx-publishing.git#egg=cnx-publishing
+-e git+https://github.com/Connexions/cnx-db.git#egg=cnx-db
+
```

Comment:
Ah, yup, I did'nt see that. The presence of the version req. lower down confused me.